packages/nimble-components/src/wafer-map/modules/computations.module.ts

```diff
@@ -0,0 +1,189 @@
+import { scaleBand, ScaleLinear, scaleLinear } from 'd3-scale';
+import type { WaferMapDie } from '../types';
+import { Dimensions, Margin, WaferMapQuadrant } from '../types';
+
+interface MapDimensions {
+    origin: {
+        x: number,
+        y: number
+    };
+    rows: number;
+    cols: number;
+}
+
+/**
+ * Computations module
+ */
```

Comment:
Can you provide a better description for developers? Maybe something like:

```suggestion
/**
 * Computations calculates the sizing for parts of a Wafer Map
 */
```